bioptim/dynamics/dynamics_functions.py

```diff
@@ -264,7 +294,7 @@ def torque_derivative_driven(
             dxdt[nlp.states["qdot"].index, :] = ddq
             dxdt[nlp.states["tau"].index, :] = horzcat(*[dtau for _ in range(ddq.shape[1])])
 
-        return dxdt
+        return dxdt, MX(nlp.states.shape, ddq.shape[1])
```

Comment:
Does this actually work, or the MX is a place holder? If so, please make sure we raise an `NotImplementedError` somewhere so noone tries to use this dynamic function

Reply:
The flag is hidden in the collocation integrator, it allows to switch from the new implicit constraint to the older one when MX=0 is detected.